### PR TITLE
Misc cleanup tasks

### PR DESCRIPTION
**Pull request by gfontenot**
Just fixing up a few things stylistically that were bothering me.

### CODE REVIEW

File: Argo/Globals/JSONValue.swift [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
 
-    case let v as [String:AnyObject]:
-      var object: [String:JSONValue] = [:]
-      for key in v.keys {
-        if let value: AnyObject = v[key] {
-          object[key] = parse(value)
-        } else {
-          object[key] = .JSONNull
-        }
+    case let v as [String: AnyObject]:
```
Comment (tonyd256) [inline Argo/Globals/JSONValue.swift]: `as JSONDict` maybe? but i can also see you liking the full type here ... either way I'm ok

Comment (gfontenot) [inline Argo/Globals/JSONValue.swift]: `JSONDict` is `[String: JSONValue]`. This is `[String: AnyObject]`. I think it's actually just an `NSDictionary` that's being bridged to `Dictionary`.

Comment (tonyd256) [inline Argo/Globals/JSONValue.swift]: yup right nvm

**Comment by tonyd256** [comment]
:+1:

**Pull request merged**